app/src/main/java/de/westnordost/streetcomplete/quests/summit_register/AddSummitRegister.kt

```diff
@@ -48,10 +48,10 @@ class AddSummitRegister : OsmElementQuestType<Boolean> {
         val hikingRoutes = mapData.relations
```

Comment:
this code is now dead and should be also removed

in general `getApplicableElements` can be simplified

main problem is now it will be now asked for peaks far away from any paths... Maybe it should be asked for path close to `highway=path` ways?

Reply:
I didn't want to remove it because it might start working again if at some point (hiking) relations are not ignored any more.

Far away from paths might not be that much of an issue if someone actually carried a cross up there?

Reply:
> if at some point (hiking) relations are not ignored any more

unlikely (it caused outrageous performance issues), and in such case old commits still exist

> Far away from paths might not be that much of an issue if someone actually carried a cross up there?

I am pretty sure that some crosses are on summits inaccessible by regular hikers

Reply:
I doubt it's worth the effort filtering out those rare inaccessible crosses, but ok.
Is it sufficient to replace `route=hiking` relations with `highway=path` ways?

Reply:
>  Is it sufficient to replace route=hiking relations with highway=path ways?

allowing also `highway=footway` and maybe `highway=track` may make sense

And it is worth testing is it murdering performance or not.

Reply:
> I doubt it's worth the effort filtering out those rare inaccessible crosses, but ok.

You can also inspect already mapped crosses and check how often they are inaccessible

Bigger benefit of asking near paths/tracks is that far more peaks that should be eligible will get this quest

Reply:
Oh, I didn't realize this was about all peaks in the `filter`, I was rather narrowly focused onto peaks with cross...
And also private paths/tracks shoulds be filtered out.

Reply:
Jup, this is why I do not think that approach is viable.

This quest is effectively dead, but I hesitate to remove it completely because it lying around doesn't hurt plus maybe one day, route relations can be included again. Then, we do not need to search deep in the git history of this project to make this quest work again.

Reply:
If `summit:cross` is not enough, how about requiring nearby paths to have `sac_scale` tagged? This should not happen in a city (well, maybe exclude `sac_scale=hiking`).

Reply:
Hmm yeah, I guess this would work. Of course it must be an inclusive list, not exclusive list

Reply:
All values in https://wiki.openstreetmap.org/wiki/Key:sac%20scale except `hiking`? Or maybe also exclude `difficult_alpine_hiking` as it includes climbing?

Reply:
I think everything except hiking is fine